salt/grains/core.py

```diff
@@ -1283,6 +1284,11 @@ def os_data():
                         grains['lsb_distrib_release'] = os_release['VERSION_ID']
                     if 'PRETTY_NAME' in os_release:
                         grains['lsb_distrib_codename'] = os_release['PRETTY_NAME']
+                    if 'CPE_NAME' in os_release:
+                        if ":suse:" in os_release['CPE_NAME']:
+                            grains['os'] = "SUSE"
+                        elif ":opensuse:" in os_release['CPE_NAME']:
+                            grains['os'] = "SUSE"
```

Comment:
It is too much noise for _setting_ the `os` key. I would put that into one clause:
```python
if ":suse:" in os_release['CPE_NAME'] or ":opensuse:" in os_release['CPE_NAME']:
    grains['os'] = "SUSE"
```